### ISSUE TITLE
Need clear instructions on installing from scratch

### ISSUE THREAD

**Issue by arjunnath** [linkedin/oncall#175]
Hi,

I'm having trouble installing oncall in our linux environment using the instructions given.

Here is one of the errors while running **"python setup.py develop"** : 

`
Processing dependencies for oncall==1.1.0                 
Searching for slackclient                                 
Reading https://pypi.python.org/simple/slackclient/       
Downloading https://files.pythonhosted.org/packages/1f/9e/dd5bb327786be753e5d8de976082eb714cb83abeb00f67f0646ba2e166ad/slackclient-1.2.1.tar.gz#sha256=e9de0c893e8c5473107f5927ae1e543d35246f0c834f5e86470b22b762211577               
error: Download error for https://files.pythonhosted.org/packages/1f/9e/dd5bb327786be753e5d8de976082eb714cb83abeb00f67f0646ba2e166ad/slackclient-1.2.1.tar.gz#sha256=e9de0c893e8c5473107f5927ae1e543d35246f0c834f5e86470b22b762211577: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:777)
`
I have a venv set up with python 3.6.3

1) Can we have clear step by step instructions to download and install oncall for both development and separately for production use ? Please note that not everyone uses Docker.

2) Is it possible to package oncall so we can start it easily ? i.e. minimum run. For example in a VM or as a single zip file ?

Thanks

**Comment by dwang159** [comment]
This appears to be a problem with SSL verification on files.pythonhosted.org; probably transient. We have installation instructions for development in our Readme, and Docker for quick setup. Support for services other than Docker is something we'd like to have in the future, and if you'd like to contribute an integration for k8s, AWS, or something else, that would be awesome.

**Comment by arjunnath** [comment]
> Support for services other than Docker is something we'd like to have in the future

I meant simple install and start for bare-metal hosts and VMs.
Also some instructions for running a production instance. All the info seems to be development oriented.

**Comment by jeremybender-work** [comment]
@foequeue Go for it!  I doubt anyone would shred it.  I'm definitely not a programmer, but have had a couple of minor changes committed with absolutely no shredding.

**Comment by arjunnath** [comment]
Yes, it would be good to have some explanation of the installation process. Also integration with AD/LDAP.

**Comment by easyaayush** [comment]
> Yes, it would be good to have some explanation of the installation process. Also integration with AD/LDAP.

Did you find any explanation? please share with me if you have.

**Comment by arjunnath** [comment]
> > Yes, it would be good to have some explanation of the installation process. Also integration with AD/LDAP.
> 
> Did you find any explanation? please share with me if you have.

Unfortunately, we didn't get any method and so we  built our own internal app in the end.
